### ISSUE TITLE
[Accessibility]: Change search input in header from `'combobox` to `searchinput`

### ISSUE THREAD

**Issue by mnorthuis** [department-of-veterans-affairs/vets-design-system-documentation#5887]
### Point of contact

@coforma-jamie 

### How severe do you think this issue is?

Medium

### Severity rationale

Screen reader users may be prompted to control this box as a combo box rather than an input.

### URL where issue occurs

www.va.gov

### Component or pattern (if applicable)

Header

### Environment details

Any OS or browser.  

### Steps to reproduce

View header on VA.gov. 

### Observed behavior

The autosuggest feature has been disabled for search.  The header search box still reflects the `combobox` functionality and should be changed to a `searchinpu`t field.

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [ ] Color, typography, & visual elements
- [x] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [ ] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else
